emit_translations.py

```diff
@@ -44,13 +44,25 @@ def translate_src_files(src_dir: str, out_dir: str, translations: dict[Macro, st
         startLine = int(startDefLocParts[1]) - 1
         endLine = int(endDefLocParts[1]) - 1
 
+        endLineContent = src_file_content[endLine]
+
+        # Some code bases may define macros with an opening comment on the last line,
+        # preserve it here
+        if '/*' in endLineContent.strip() and '*/' not in endLineContent.strip():
```

Comment:
Yeah this is definitely a hack but should work in most cases; let's just hope we don't see something like this

```
#define FOO "/*"
```

We can leave this as-is for now and address later if we need a more robust solution

Reply:
Definitely - added a todo comment about finding a better way to parse down the line